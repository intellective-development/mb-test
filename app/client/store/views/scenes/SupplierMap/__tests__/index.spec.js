/* eslint import/first: 0 */
jest.mock('client/shared/components/higher_order/make_provided'); // avoid importing the full store

import * as React from 'react';
import supplier_factory from '@minibar/store-business/src/supplier/__tests__/supplier.factory';

import TestProvider from 'store/views/__tests__/utils/TestProvider';
import { SupplierMapModal } from '../index';

describe('SupplierMapModal', () => {
  const supplier_id = 1;
  const supplier = supplier_factory.build('with_delivery_methods', {id: supplier_id});

  it('renders', () => {
    const state = {
      ...supplier_factory.stateify([supplier]),
      ui: {
        map_supplier_id: supplier_id
      }
    };
    expect(render(
      <TestProvider initial_state={state}>
        <SupplierMapModal supplier_id={supplier_id} show />
      </TestProvider>
    )).toMatchSnapshot();
  });
});
