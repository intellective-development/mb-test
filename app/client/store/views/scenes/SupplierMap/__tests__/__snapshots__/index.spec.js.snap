// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupplierMapModal renders 1`] = `null`;
